index.html

```diff
@@ -4,10 +4,10 @@
 <head>
     <meta charset="UTF-8">
     <title>Lotto</title>
-    <link rel="stylesheet" href="../lottospiel/main.css">
-    <link rel="stylesheet" href="../lottospiel/lottospiel.css">
-    <script src="../lottospiel/lib.js"></script>
-    <script src="../lottospiel/lottospiel.js"></script>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/lottospiel.css">
+    <script src="js/lib.js"></script>
+    <script src="js/lottospiel.js"></script>
```

Comment:
This unfortunately won't work, because the CSS and JS files are not located in `css` and `js` subdirectories. Either move them to such subdirectories, or change the paths.